include/fixed_containers/enum_array.hpp

```diff
@@ -69,9 +70,16 @@ class EnumArray
 
     constexpr EnumArray(std::initializer_list<std::pair<const L, T>> list) noexcept
         requires DefaultConstructible<T>
+      : EnumArray(std_transition::from_range, list)
+    {
+    }
+
+    template <class R>
```

Comment:
im not very knowlegeable about value semantics but I had a question:
shouldnt the for loop be
```
for (auto &&[label,value] : std::forward<R>(rg))
```
 to preserve the valueness? in the current state rg is passed in as a universal reference but converted to an lvalue in the for loop. @alexkaratarakis

Reply:
Linking to PR with discussion: https://github.com/teslamotors/fixed-containers/pull/82